[2023-04-18T12:56:27.807+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: process-employees.create_employees_table scheduled__2023-04-17T00:00:00+00:00 [queued]>
[2023-04-18T12:56:27.821+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: process-employees.create_employees_table scheduled__2023-04-17T00:00:00+00:00 [queued]>
[2023-04-18T12:56:27.821+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-04-18T12:56:27.821+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-04-18T12:56:27.821+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-04-18T12:56:27.860+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_employees_table> on 2023-04-17 00:00:00+00:00
[2023-04-18T12:56:27.871+0000] {standard_task_runner.py:55} INFO - Started process 5031 to run task
[2023-04-18T12:56:27.874+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'process-employees', 'create_employees_table', 'scheduled__2023-04-17T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/process-employees.py', '--cfg-path', '/tmp/tmpkr5ihtra']
[2023-04-18T12:56:27.874+0000] {standard_task_runner.py:83} INFO - Job 9: Subtask create_employees_table
[2023-04-18T12:56:27.960+0000] {task_command.py:388} INFO - Running <TaskInstance: process-employees.create_employees_table scheduled__2023-04-17T00:00:00+00:00 [running]> on host cbbda7f0b94f
[2023-04-18T12:56:28.062+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=process-employees
AIRFLOW_CTX_TASK_ID=create_employees_table
AIRFLOW_CTX_EXECUTION_DATE=2023-04-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-17T00:00:00+00:00
[2023-04-18T12:56:28.063+0000] {sql.py:254} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS employees (
                "Serial Number" NUMERIC PRIMARY KEY,
                "Company Name" TEXT,
                "Employee Markme" TEXT,
                "Description" TEXT,
                "Leave" INTEGER
            );
[2023-04-18T12:56:28.076+0000] {base.py:73} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2023-04-18T12:56:28.375+0000] {base.py:73} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2023-04-18T12:56:28.379+0000] {sql.py:375} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS employees (
                "Serial Number" NUMERIC PRIMARY KEY,
                "Company Name" TEXT,
                "Employee Markme" TEXT,
                "Description" TEXT,
                "Leave" INTEGER
            );, parameters: None
[2023-04-18T12:56:28.384+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 266, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "pg_type_typname_nsp_index"
DETAIL:  Key (typname, typnamespace)=(employees, 2200) already exists.

[2023-04-18T12:56:28.393+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=process-employees, task_id=create_employees_table, execution_date=20230417T000000, start_date=20230418T125627, end_date=20230418T125628
[2023-04-18T12:56:28.406+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 9 for task create_employees_table (duplicate key value violates unique constraint "pg_type_typname_nsp_index"
DETAIL:  Key (typname, typnamespace)=(employees, 2200) already exists.
; 5031)
[2023-04-18T12:56:28.447+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-04-18T12:56:28.519+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-04-18T13:04:33.384+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: process-employees.create_employees_table scheduled__2023-04-17T00:00:00+00:00 [queued]>
[2023-04-18T13:04:33.396+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: process-employees.create_employees_table scheduled__2023-04-17T00:00:00+00:00 [queued]>
[2023-04-18T13:04:33.396+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-04-18T13:04:33.396+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-04-18T13:04:33.396+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-04-18T13:04:33.414+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_employees_table> on 2023-04-17 00:00:00+00:00
[2023-04-18T13:04:33.424+0000] {standard_task_runner.py:55} INFO - Started process 5953 to run task
[2023-04-18T13:04:33.427+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'process-employees', 'create_employees_table', 'scheduled__2023-04-17T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/process-employees.py', '--cfg-path', '/tmp/tmpwz56t5nh']
[2023-04-18T13:04:33.427+0000] {standard_task_runner.py:83} INFO - Job 14: Subtask create_employees_table
[2023-04-18T13:04:33.491+0000] {task_command.py:388} INFO - Running <TaskInstance: process-employees.create_employees_table scheduled__2023-04-17T00:00:00+00:00 [running]> on host cbbda7f0b94f
[2023-04-18T13:04:33.564+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=process-employees
AIRFLOW_CTX_TASK_ID=create_employees_table
AIRFLOW_CTX_EXECUTION_DATE=2023-04-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-17T00:00:00+00:00
[2023-04-18T13:04:33.564+0000] {sql.py:254} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS employees (
                "Serial Number" NUMERIC PRIMARY KEY,
                "Company Name" TEXT,
                "Employee Markme" TEXT,
                "Description" TEXT,
                "Leave" INTEGER
            );
[2023-04-18T13:04:33.575+0000] {base.py:73} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2023-04-18T13:04:33.829+0000] {base.py:73} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2023-04-18T13:04:33.834+0000] {sql.py:375} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS employees (
                "Serial Number" NUMERIC PRIMARY KEY,
                "Company Name" TEXT,
                "Employee Markme" TEXT,
                "Description" TEXT,
                "Leave" INTEGER
            );, parameters: None
[2023-04-18T13:04:33.847+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=process-employees, task_id=create_employees_table, execution_date=20230417T000000, start_date=20230418T130433, end_date=20230418T130433
[2023-04-18T13:04:33.879+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-04-18T13:04:33.914+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-04-18T13:14:36.369+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: process-employees.create_employees_table scheduled__2023-04-17T00:00:00+00:00 [queued]>
[2023-04-18T13:14:36.382+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: process-employees.create_employees_table scheduled__2023-04-17T00:00:00+00:00 [queued]>
[2023-04-18T13:14:36.382+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-04-18T13:14:36.383+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-04-18T13:14:36.383+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-04-18T13:14:36.400+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_employees_table> on 2023-04-17 00:00:00+00:00
[2023-04-18T13:14:36.452+0000] {standard_task_runner.py:55} INFO - Started process 7096 to run task
[2023-04-18T13:14:36.456+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'process-employees', 'create_employees_table', 'scheduled__2023-04-17T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/process-employees.py', '--cfg-path', '/tmp/tmp6wrbgl4d']
[2023-04-18T13:14:36.456+0000] {standard_task_runner.py:83} INFO - Job 22: Subtask create_employees_table
[2023-04-18T13:14:36.556+0000] {task_command.py:388} INFO - Running <TaskInstance: process-employees.create_employees_table scheduled__2023-04-17T00:00:00+00:00 [running]> on host cbbda7f0b94f
[2023-04-18T13:14:36.668+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=process-employees
AIRFLOW_CTX_TASK_ID=create_employees_table
AIRFLOW_CTX_EXECUTION_DATE=2023-04-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-17T00:00:00+00:00
[2023-04-18T13:14:36.669+0000] {sql.py:254} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS employees (
                "Serial Number" NUMERIC PRIMARY KEY,
                "Company Name" TEXT,
                "Employee Markme" TEXT,
                "Description" TEXT,
                "Leave" INTEGER
            );
[2023-04-18T13:14:36.679+0000] {base.py:73} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2023-04-18T13:14:36.954+0000] {base.py:73} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2023-04-18T13:14:36.958+0000] {sql.py:375} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS employees (
                "Serial Number" NUMERIC PRIMARY KEY,
                "Company Name" TEXT,
                "Employee Markme" TEXT,
                "Description" TEXT,
                "Leave" INTEGER
            );, parameters: None
[2023-04-18T13:14:36.973+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=process-employees, task_id=create_employees_table, execution_date=20230417T000000, start_date=20230418T131436, end_date=20230418T131436
[2023-04-18T13:14:37.028+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-04-18T13:14:37.077+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
